examples/website-demos/virtual-event/src/components/CentralPanel/EventCard.tsx

```diff
@@ -27,9 +28,7 @@ export const EventCard: React.FC<Props> = (props) => {
 
   return (
     <div className='event-card-container' onClick={handleClick}>
-      <div className='event-card-image'>
-        <Image />
-      </div>
+      <div className='event-card-image'>{Image && <Image />}</div>
```

Comment:
evaluate the Image conditional on the left side of the div, otherwise you're passing `false` to the div if it doesnt exist